languages/scala/exercises/concept/operators/.docs/instructions.md

```diff
@@ -0,0 +1,60 @@
+In this exercise you're going to write a calculator for rectangles.
+
+Given is a case class `Rectangle`. A rectangle is defined by its width and its height:
+
+```scala
+case class Rectangle(width: Int, height: Int)
+```
+
+The calculator will allow to sum up two rectangles and to increase a rectangle by multiplying it with a number.
+
+## 1. Create a plus operator for rectangles
+
+Implement a `+` method that takes another rectangle as argument. Assume that the passed rectangle has the same height as the current one. The method should return a new rectangle, whose width is a sum of both rectangle's widths and whose height is the same as of both rectangles.
+
+Let's visualize the expected outcome:
+
+![sum of 2 rectangles of same height](sum_same_height.png)
+
+This code snippet shows how to use the `+` method:
+
+```scala
+val leftRectangle = Rectangle(2, 5)
+val rightRectangle = Rectangle(1, 5)
+val sumRectangle = leftRectangle + rightRectangle
+println(sumRectangle)
+// => Rectangle(3, 5)
+```
+
+## 2. Enhance the plus operator
```

Comment:
Similar to the comment on previous task, this could maybe be phrased more in terms of _what_ to do instead instead of _how_. Maybe something like:

```suggestion
## 2. Allow adding rectangles of different heights
```